### PR TITLE
Ca 178 add score notion to compliance assessment

### PR DESCRIPTION
**Pull request by Mohamed-Hacene**

### CODE REVIEW

**Comment by eric-intuitem** [comment]
Current version is very slow.

**Comment by ab-smith** [comment]
Contributing to #193

**Comment by ab-smith** [comment]
to fix #193

File: backend/core/serializers.py [hunk 1]
```diff
@@ -499,6 +498,30 @@ class Meta:
 
 
 class RequirementAssessmentWriteSerializer(BaseModelSerializer):
+    def validate_score(self, value):
+        framework = self.get_framework()
+
+        if value is not None:
+            if value < framework.min_score or value > framework.max_score:
+                raise serializers.ValidationError(
+                    f"Score must be between {framework.min_score} and {framework.max_score}"
```
Comment (Mohamed-Hacene) [inline backend/core/serializers.py]: We need to let `{score: f"...}` at the beginning to indicate field error on frontend

**Review by eric-intuitem — APPROVED** [review]
LGTM.
We could calculate intermediate aggregate scores on frontend side, but this is not mandatory for the first version.

**Review by eric-intuitem — APPROVED** [review]
LGTM

**Pull request merged**